tools/scripts/octf-setup-cmake.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# Copyright(c) 2012-2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# This script dir name
+OCTF_SCRIPT_DIR="$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)"
+. ${OCTF_SCRIPT_DIR}/octf-common.sh
+
+OCTF_CMAKE_MIN_VER_MAJOR=3
+OCTF_CMAKE_MIN_VER_MINOR=10
+OCTF_CMAKE_MIN_VER=${OCTF_CMAKE_MIN_VER_MAJOR}.${OCTF_CMAKE_MIN_VER_MINOR}
+
+function cmake_info ()
+{
+    info "Minimal cmake version is ${OCTF_CMAKE_MIN_VER}"
+}
+
+function is_cmake_version_ok ()
+{
+    command -v cmake &>/dev/null
+    if [ $? -ne 0 ]
+    then
+        return 1
+    fi
+
+    local prompt="cmake version "
+    local version=$(cmake --version | grep "${prompt}" | sed -e s/^"${prompt}"//)
+
+    if [ "${version}" == "" ]
+    then
+        echo b
+        return 1
+    fi
+
+    local major=$(echo ${version} | awk -F '.' '{print $1}' )
+    local minor=$(echo ${version} | awk -F '.' '{print $2}' )
+
+    if [ "${major}" == "" ] || [ "${minor}" == "" ]
+    then
+        echo c
+        return 1
+    fi
+
+    if [ ${major} -lt 3 ] || [ ${minor} -lt 10 ]
```

Comment:
i think this should be 
major < OCTF_CMAKE_MIN_VER_MAJOR || (major == OCTF_CMAKE_MIN_VER_MAJOR && minor < OCTF_CMAKE_MIN_VER_MINOR)